packages/codegen-ui/lib/generate-form-definition/form-to-component.ts

```diff
@@ -106,7 +119,7 @@ export const ctaButtonConfig = (): StudioComponentChild => {
         name: 'CancelButton',
         properties: {
           label: {
-            value: 'Cancel',
+            value: cancelLabel,
```

Comment:
just noticed, also, that you would want to pass in variants to get those differences in style